runtime/libs/tflite/src/OutputResetter.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "tflite/OutputResetter.h"
+#include "tflite/TensorView.h"
+
+#include <misc/tensor/IndexIterator.h>
+
+namespace nnfw
+{
+namespace tflite
+{
+
+void OutputResetter::run(::tflite::Interpreter &interp)
+{
+  for (const auto &tensor_idx : interp.outputs())
+  {
+    TfLiteTensor *tensor = interp.tensor(tensor_idx);
+    if (tensor->type == kTfLiteInt32)
+    {
+      resetValue<int32_t>(interp, tensor_idx);
+    }
+    else if (tensor->type == kTfLiteUInt8)
+    {
+      resetValue<uint8_t>(interp, tensor_idx);
+    }
+    else if (tensor->type == kTfLiteInt8)
+    {
+      resetValue<int8_t>(interp, tensor_idx);
+    }
+    else if (tensor->type == kTfLiteBool)
+    {
+      resetValue<bool>(interp, tensor_idx);
+    }
+    else
+    {
+      assert(tensor->type == kTfLiteFloat32);
```

Comment:
@hseok-oh Before this commit, output resetter throws unsupported type exeception when it does not matched any types it supports. Currently, `else` assumes `float32` and check via `assert`. If it is sure by some prior-conditions that type could not have other type. It is okay. However, I think the previous way (explicit runtime throw) is safer.

Reply:
Good. I'll update.